modules/modules.nix

```
@@ -96,6 +96,7 @@ let
     (loadModule ./services/flameshot.nix { })
     (loadModule ./services/gnome-keyring.nix { })
     (loadModule ./services/gpg-agent.nix { })
+    (loadModule ./services/ihaskell.nix { })
```

Since this module uses systemd it should be limited to Linux:
```suggestion
    (loadModule ./services/ihaskell.nix { condition = hostPlatform.isLinux; })
```